### PR TITLE
(TNL-60) Listed maximum in numerical input description paragraph

### PR DESCRIPTION
**Pull request by marjev**
JIRA bug report: https://openedx.atlassian.net/browse/TNL-60

@cahrens I propose this as a solution for TNL-60. Could you and/or someone else from TNL team provide a review?

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @marjev! It looks like you're a member of a company that does contract work for edX. If you're doing this work as part of a paid contract with edX, you should talk to edX about who will review this pull request. If this work is not part of a paid contract with edX, then you should ensure that there is an OSPR issue to track this work in [JIRA](https://openedx.atlassian.net), so that we don't lose track of your pull request. 

To automatically create an OSPR issue for this pull request, just visit this link: https://openedx-webhooks.herokuapp.com/github/process_pr?repo=edx%2Fedx-ora2&number=713

**Comment by cahrens** [comment]
@symbolist can someone from your team review this very small PR?

**Comment by symbolist** [comment]
@catong A few strings have been modified in this PR. Can you review them?

File: openassessment/templates/openassessmentblock/edit/oa_edit.html [hunk 1]
```diff
@@ -132,7 +132,7 @@ <h2 class="openassessment_alert_title">{% trans "Rubric Change Impacts Settings
                             min="0" max="99"
                         />
                     </div>
-                    <p class="setting-help">{% trans "Specify the number of top-scoring responses to display. If the number is 0, the Top Responses section won't appear to students." %}</p>
+                    <p class="setting-help">{% trans "Specify the number of top-scoring responses to display. If the number is 0, the Top Responses section won't appear to students. Maximum: 99" %}</p>
```
Comment (catong) [inline openassessment/templates/openassessmentblock/edit/oa_edit.html]: @symbolist 
"Specify the number of top scoring responses to display. Valid numbers are 0 to 99. If the number is 0, the Top Responses section does not appear to learners."

File: openassessment/templates/openassessmentblock/edit/oa_edit_peer_assessment.html [hunk 2]
```diff
@@ -22,14 +22,14 @@
                         <label for="peer_assessment_must_grade" class="setting-label">{% trans "Must Grade" %}</label>
                         <input id="peer_assessment_must_grade" class="input setting-input" type="number" value="{{ assessments.peer_assessment.must_grade }}" min="0" max="99">
                     </div>
-                    <p class="setting-help">{% trans "Specify the number of peer assessments that each student must complete."%}</p>
+                    <p class="setting-help">{% trans "Specify the number of peer assessments that each student must complete. Maximum: 99"%}</p>
```
Comment (catong) [inline openassessment/templates/openassessmentblock/edit/oa_edit_peer_assessment.html]: "Specify the number of peer assessments that each learner must complete. Valid numbers are 0 to 99."

File: openassessment/templates/openassessmentblock/edit/oa_edit_peer_assessment.html [hunk 3]
```diff
@@ -22,14 +22,14 @@
...
                 </li>
                 <li class="field comp-setting-entry">
                     <div class="wrapper-comp-setting">
                         <label for="peer_assessment_graded_by" class="setting-label"> {% trans "Graded By" %}</label>
                         <input id="peer_assessment_graded_by" class="input setting-input" type="number" value="{{ assessments.peer_assessment.must_be_graded_by }}" min="0" max="99">
                     </div>
-                    <p class="setting-help">{% trans "Specify the number of students who must assess each response."%}</p>
+                    <p class="setting-help">{% trans "Specify the number of students who must assess each response. Maximum: 99"%}</p>
```
Comment (catong) [inline openassessment/templates/openassessmentblock/edit/oa_edit_peer_assessment.html]: @symbolist 
"Specify the number of learners that each response must be assessed by. Valid numbers are 0 to 99."

**Comment by catong** [comment]
@symbolist My suggestions in inline comments. Thanks.

**Comment by symbolist** [comment]
@marjev Can you please update the strings?

**Comment by marjev** [comment]
@symbolist I've updated the strings as @catong suggested.
Thanks @catong!

**Comment by symbolist** [comment]
:+1:

**Comment by catong** [comment]
@marjev Looks good. Thanks! :+1:

**Pull request merged**